website_v2/gcloud-powershell-website-v2/www/static/app/referenceController.js

```diff
@@ -0,0 +1,23 @@
+﻿var app = angular.module('powershellSite');
+
+/** The controller that exposes route parameters to the templates. **/
+app.controller('ReferenceController', ['$scope', '$routeParams',
+    function($scope, $routeParams) {
+        var prodInfo = $scope.productInfo;
+        if (prodInfo === null) return;
```

Comment:
So this is good that nothing else happens but maybe add an message/error to the user in the template?